nemo_skills/pipeline/convert.py

```diff
@@ -298,12 +298,22 @@ def convert(
     cluster_config = get_cluster_config(cluster, config_dir)
     cluster_config = resolve_mount_paths(cluster_config, mount_paths)
 
-    input_model, output_model, log_dir = check_mounts(
-        cluster_config,
-        log_dir=log_dir,
-        mount_map={input_model: "/input_model", output_model: "/output_model"},
-        check_mounted_paths=check_mounted_paths,
-    )
+    if convert_from == "hf" and not input_model.startswith("/"):
```

Comment:
does it actually work? As in does the conversion script download the model properly or does it fail? I remember some issues, but maybe I'm wrong

Reply:
yes, it worked. I fixed it.

Reply:
Removed these lines - 
https://github.com/NVIDIA/NeMo-Skills/blob/main/nemo_skills/conversion/hf_to_nemo_llama.py#L138